assets/app/scripts/services/logLinks.js

```diff
@@ -50,10 +50,28 @@ angular.module('openshiftConsole')
         newTab(params);
       };
 
-      // Not currently used.
-      // new tab: path/to/current?view=textonly
-      var textOnlyLink = function() {
-        newTab({view: 'textonly'});
+      // broken up for readability:
+      var template = new URITemplate([
+        "/#/discover?",
+        "_g=(",
+          "time:(from:now-7y%2Fy,mode:relative,to:now)",
+        ")",
+        "&_a=(",
+          "columns:!(_source),",
+          "index:'{namespace}.*',",
+          "query:(",
+            "query_string:(",
+              "analyze_wildcard:!t,",
+              "query:'kubernetes_pod_name: {podname} %26%26 kubernetes_namespace_name: {namespace}'",
+            ")",
+          "),",
+          "sort:!(time,desc)",
+        ")",
+        "&console_back_link={backlink}"
+      ].join(''));
```

Comment:
Been tinkering with this a good bit. Can't get the `console_container_name` or the `console_back_url` to work properly, per [the docs here](https://github.com/openshift/origin-kibana#current-supported-fragment-parameters).  Its supposed to strip anything with a leading `console_` from the hash & update UI, but I'm not seeing it.

@ewolinetz @jcantrill if you have thoughts, Ill take them!